README.md

```diff
@@ -94,14 +95,32 @@ module.exports = EngineAddon.extend({
 });
 ```
 
+Within your engine's `config` directory, create a new `environment.js` file:
+
+```js
+/*jshint node:true*/
+'use strict';
+
+module.exports = function(environment) {
+  var ENV = {
+    modulePrefix: 'ember-blog',
+    environment: environment
+  }
+
+  return ENV;
+};
+```
+
 Within your engine's `addon` directory, add a new `engine.js` file:
 
 ```js
 import Engine from 'ember-engines/engine';
-import Resolver from 'ember-engines/resolver';
+import Resolver from 'ember-engines/resolver'; // <=== IMPORTANT - custom resolver!!!
```

Comment:
As a complete aside, we should probably try to help folks deal with this "trollage" a bit and assert if the app isn't using our resolver.